haas/api.py

```diff
@@ -529,24 +626,24 @@ def network_create(network, creator, access, net_id):
     auth_backend = get_auth_backend()
     _assert_absent(model.Network, network)
 
-    # Check authorization and legality of arguments, and find correct 'access' and 'creator'
-    if creator != "admin":
-        creator = _must_find(model.Project, creator)
-        auth_backend.require_project_access(creator)
+    # Check authorization and legality of arguments, and find correct 'access' and 'owner'
+    if owner != "admin":
+        owner = _must_find(model.Project, owner)
+        auth_backend.require_project_access(owner)
         # Project-owned network
-        if access != creator.label:
-            raise BadArgumentError("Project-created networks must be accessed only by that project.")
+        if access != owner.label:
+            raise BadArgumentError("Project-owned networks must be accessed only by that project.")
```

Comment:
I wasn't sure how to handle passing in a single project or optionally a list so the way it currently works is that the initial access for project created networks must be that project then other projects can be added once the network is created